cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -1,151 +1,151 @@
+- ../../../../base/core/configmaps/user-workload-monitoring-config
+- ../../../../base/core/namespaces/acme-operator
+- ../../../../base/core/namespaces/aicoe-meteor
+- ../../../../base/core/namespaces/apicurio-apicurio-registry
+- ../../../../base/core/namespaces/b4mad-racing
+- ../../../../base/core/namespaces/debezium-ui
+- ../../../../base/core/namespaces/democratic-csi
+- ../../../../base/core/namespaces/dex
+- ../../../../base/core/namespaces/ds-github-labeler
+- ../../../../base/core/namespaces/ds-ml-workflows-ws
+- ../../../../base/core/namespaces/fybrik-applications
+- ../../../../base/core/namespaces/fybrik-blueprints
+- ../../../../base/core/namespaces/fybrik-system
+- ../../../../base/core/namespaces/koku-metrics-operator
+- ../../../../base/core/namespaces/neu-cs6620
+- ../../../../base/core/namespaces/openshift-cnv
+- ../../../../base/core/namespaces/openshift-logging
+- ../../../../base/core/namespaces/openshift-nfd
+- ../../../../base/core/namespaces/openshift-operators
+- ../../../../base/core/namespaces/openshift-pipelines
+- ../../../../base/core/namespaces/openshift-vertical-pod-autoscaler
+- ../../../../base/core/namespaces/opf-alertreceiver
+- ../../../../base/core/namespaces/opf-argo
+- ../../../../base/core/namespaces/opf-ci-pipelines
+- ../../../../base/core/namespaces/opf-ci-prow
+- ../../../../base/core/namespaces/opf-ci-test-pods
+- ../../../../base/core/namespaces/opf-google-spark-operator
+- ../../../../base/core/namespaces/opf-jupyterhub
+- ../../../../base/core/namespaces/opf-jupyterhub-stage
+- ../../../../base/core/namespaces/opf-kafka
+- ../../../../base/core/namespaces/opf-monitoring
+- ../../../../base/core/namespaces/opf-observatorium
+- ../../../../base/core/namespaces/opf-pulp
+- ../../../../base/core/namespaces/opf-seldon
+- ../../../../base/core/namespaces/opf-dashboard
+- ../../../../base/core/namespaces/opf-slack-first
+- ../../../../base/core/namespaces/opf-superset
+- ../../../../base/core/namespaces/opf-triage-party
+- ../../../../base/core/namespaces/opf-trino
+- ../../../../base/core/namespaces/opf-trino-stage
+- ../../../../base/core/namespaces/prometheus-anomaly-detector
+- ../../../../base/core/namespaces/report-system
+- ../../../../base/core/namespaces/seraph-ingress
+- ../../../../base/core/namespaces/seraph-platform-mq
+- ../../../../base/core/namespaces/tekton-chains
+- ../../../../base/core/namespaces/tekton-pipelines
+- ../../../../base/core/namespaces/thoth-amun-api-prod
+- ../../../../base/core/namespaces/thoth-amun-inspection-prod
+- ../../../../base/core/namespaces/thoth-backend-prod
+- ../../../../base/core/namespaces/thoth-bots-prod
+- ../../../../base/core/namespaces/thoth-deployment-examples
+- ../../../../base/core/namespaces/thoth-frontend-prod
+- ../../../../base/core/namespaces/thoth-graph-prod
+- ../../../../base/core/namespaces/thoth-infra-prod
+- ../../../../base/core/namespaces/thoth-middletier-prod
+- ../../../../base/core/namespaces/ushift-dev
```

Comment:
can you add the `tremor-demo` namespace in this list?

Reply:
absolutely.